libc-bottom-half/sources/preopens.c

```diff
@@ -71,15 +72,37 @@ static int resize(void) {
     return 0;
 }
 
+// Normalize an absolute path. Removes leading `/` and leading `./`, so the
+// first character is the start of a directory name. Additionally translate `.`
+// to the empty string.
```

Comment:
I actually thought that this was already how things were supposed to work where either `/foo` or `foo` worked fine! But it's true this would be good to document the default of `/` somewhere user-facing...